robocopkaka/bootcamp-29-project
# #158266234 Fix uncaught error in UserProfile component

#### What does this PR do?
fix uncaught error that comes up when a user goes to their profile
#### Description of Task to be completed?
* remove `eventId` from initial internal state
#### How should this be manually tested?
* clone this repo
* cd into the project folder
* run `npm install` to install all dependecies
* run `npm run start:all` to start up the application
* go to `localhost:8000` on your browser
* click on `Signup`
* enter your details
* click on `Signup`
* click on `Login`
* enter your details
* click on `Login`
* click on the `Profile` dropdown
* click on `Profile`
* open the console on your browser
* you should not see an `uncaught error` in it
#### Any background context you want to provide?
users would get an uncaught error in their console anytime they went to their profiles prior to this PR being raised
#### What are the relevant pivotal tracker stories?(if applicable)
#158266234
